library/src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -311,7 +366,7 @@ public void setLongitude(double value) {
 	 * @throws IllegalStateException if location has not been set
 	 */
 	public Map getMap() {
-		if (this.latitude == Double.NaN || this.longitude == Double.NaN) {
```

Comment:
Should keep this. As there is still a single setter for longitude. Unless you plan to set the single setters for any location related value to private.